exercism/python/pangram/pangram.py

```diff
@@ -1,3 +1,111 @@
+# Initial Solution - Use a string
+from string import ascii_lowercase as lowercase_letters
+
+
+def is_pangram(sentence):
+    letters = ""
+
+    for char in sentence.lower():
+        if char.isalpha() and char not in letters:
+            letters += char
+    return len(letters) == len(lowercase_letters)
```

Comment:
Perhaps it's better to sort `letters` and then compare it to `lowercase_letters`...

Reply:
I don't believe you can sort a string in Python -- w/o turning the string into a list then back to a string?

Reply:
You're right, disregard this comment